app/mainscreen/components/ViewTitle.tsx

```diff
@@ -0,0 +1,22 @@
+import PageTitle from "@/app/components/PageTitle";
```

Comment:
How is this file any different from PageTitle?

Reply:
It has the score display,
right now we haven't merged it to the main branch but one you will it will dry the title for each section in the main screen as it have the title and score